Norwegian translation

erikbra opened the pull request:
Norwegian translation, based on English and Danish translation.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/ploeh/Numsense?pullRequest=30) <br/>All committers have accepted the CLA.

erikbra commented:
@ploeh, I tried signing the CLA, but it doesn't seem to have kicked in. Do I need to do anything other than accepting it?

ploeh commented:
Thank you for your interest in contributing to Numsense :+1: 

I had to dig a little in order to figure out why the CLA status is _unsigned_, but I think I've figured it out. The commit in this pull request was authored by _erikbran_, whom, I believe, is you, but not the same identity as @erikbra. I think you need to sign the CLA as _erikbran_, or alternatively update the commit author to be @erikbra.

erikbra commented:
Sorry for that. I have multiple git identities, one for work related, one for private. I had forgotten to update the user.name in the git config. I have now updated the commit author, and rebased the latest changes from master. the CLA check looks good now.

ploeh commented on Numsense.UnitTests/NorwegianExamples.fs:
```diff
@@ -0,0 +1,245 @@
+//[<InlineData("tomilliardertjuenimillionertohunderogsekstitusen",   2029260000)>]
+//[<InlineData(
+//    "tomilliarderetthundreogførtisjumillionerfirehundreogåttitretusensekshundreogførtisju",
+//    System.Int32.MaxValue)>]
```
Why are so many of these examples commented out?

erikbra replied:
Sorry, that was just a mishap. I'll look into it.
